oci/spec_opts_unix.go

```diff
@@ -337,11 +337,12 @@ func WithUser(userstr string) SpecOpts {
 }
 
 // WithUIDGID allows the UID and GID for the Process to be set
-func WithUIDGID(uid, gid uint32) SpecOpts {
+func WithUIDGID(uid, gid uint32, sgids []uint32) SpecOpts {
```

Comment:
I would leave this function signature alone and make a new Opts to not break existing clients.

Reply:
Fixed.